packages/ses/src/error/tame-error-constructor.js

```diff
@@ -37,9 +40,66 @@ export default function tameErrorConstructor(
 ) {
   const ErrorPrototype = FERAL_ERROR.prototype;
 
-  const { captureStackTrace: originalCaptureStackTrace } = FERAL_ERROR;
-  const platform =
-    typeof originalCaptureStackTrace === 'function' ? 'v8' : 'unknown';
+  const {
+    captureStackTrace: originalCaptureStackTrace,
+    prepareStackTrace: originalPrepareStackTrace,
+  } = FERAL_ERROR;
+  let platform = 'unknown';
+  let callSiteToStringFallback;
+  if (typeof originalCaptureStackTrace === 'function') {
+    // we might be on v8
+    if (typeof originalPrepareStackTrace === 'function') {
+      // This case should not occur on v8 or any other platform.
+      // But if it does, we assume we're on v8, or a platform whose
+      // error stack logic is close enough that we can treat it
+      // like v8.
+      platform = 'v8';
```

Comment:
I believe it does happen in Node.js according to the realm init logic I'm seeing: https://github.com/nodejs/node/blob/b8870c4a61f9c4c4490e43472e98655b00055359/lib/internal/bootstrap/realm.js#L444-L470

FYI that logic shows that it's not v8 that calls `Error.prepareStackTrace` but the embedder that sets a callback (in this case `setPrepareStackTraceCallback`), which [itself does the forwarding](https://github.com/nodejs/node/blob/b8870c4a61f9c4c4490e43472e98655b00055359/lib/internal/errors.js#L116-L156) to `Error.prepareStackTrace` (or some other fallback)